library/src/helpers/marked.ts

```diff
@@ -28,9 +29,18 @@ const markedOptions: marked.MarkedOptions = {
     }
   },
 };
+const codeHighlightRenderer = new marked.Renderer(codeHighlightOptions);
+const markdownRenderer = new marked.Renderer();
+markdownRenderer.code = (code, language, isEscaped) => {
+  if (language === 'mermaid') {
+    initializeMermaidOnce();
+    return '<pre class="mermaid">' + code + '</pre>';
```

Comment:
Dont you need to return the rendered mermaid diagram here? i.e. [add this from the website](https://github.com/asyncapi/website/blob/master/components/MDX.js#L119):
```ts
let currentId = 0;
const uuid = () => `mermaid-${(currentId++).toString()}`;
function MermaidDiagram({ graph }) {
  const [svg, setSvg] = useState(null);

  useLayoutEffect(() => {
    if (!graph) {
      return;
    }

    try {
      mermaid.mermaidAPI.render(uuid(), graph, renderedSvg => setSvg(renderedSvg));
    } catch (e) {
      setHtml(null)
      console.error(e);
    }
  }, [graph]);

  return svg ? <div dangerouslySetInnerHTML={{ __html: svg }} /> : null;
}
```

And then do the following:
```suggestion
    return <MermaidDiagram graph={code} />;
```